arenawatch.lic

```diff
@@ -1,4 +1,6 @@
-custom_require.call(%w[common common-arcana common-healing common-items common-summoning common-travel drinfomon equipmanager events spellmonitor])
+custom_require.call(%w[common common-items common-summoning common-travel drinfomon equipmanager events spellmonitor])
+
+$arena_watch_timer = nil
```

Comment:
Why is the variable needed as a global rather than local to the mainLoop method?

Reply:
Cause in my head I said, "You can't put it in a loop, it will get recreated each time." and the method name is "main loop", therefore it is a loop. (it could totally live in the mainLoop method before the loop starts)